pages/common/ldapsearch.md

```diff
@@ -0,0 +1,35 @@
+# ldapsearch
+
+> CLI utility for querying an LDAP directory.
+
+- Query an LDAP server for all items that are a member of the given group and return the object's {{displayName}} value:
+
+`ldapsearch -D '{{admin_DN}}' -w '{{password}}' -h {{ldap_host}} -b {{base_ou}} '{{memberOf}}={{group1}}' {{displayName}}`
```

Comment:
Change displayName in the description to `displayName`

Change displayName to display_name everywhere.

Also, it looks like `memberOf` is compulsory to get the member of a group. In that case, it should be outside the token.

Reply:
@agnivade: I see what you're saying, but I'm shy about changing the fields provided in the examples. `ldapsearch` works with an LDAP server's locally defined schema, so any fields provided as arguments must be treated as literal values. 

The {{memberOf}} and {{displayName}} fields are part of the default schema for Active Directory, which seems common enough to warrant being made an example.

What do you think?

Reply:
Then please remove them from the tokens.

Reply:
I think what he's trying to say is that they are concrete examples. What we might want to do is `{{memberOf=group1}}` instead of `{{memberOf}}={{group1}}`

Reply:
I drafted a change for the above. 

Honestly I seem to be having a devil of a time with formatting, so maintainers _please_ feel free to nudge any changes you think are correct.

Reply:
No problem, @lbonanomi! Yeah, our format is somewhat odd. It's based on [CommonMark](https://commonmark.org/), but it's got some customisations to it.

And in this case, I posted a comment 'cause I wanted @agnivade's feedback